rules/windows/image_load/image_load_dll_vsstrace_susp_load.yml

```
@@ -31,6 +31,7 @@ detection:
               - 'C:\Windows\SysWOW64\'
               - 'C:\Windows\Temp\{' # Installers
               - 'C:\Windows\WinSxS\'
+              - 'C:\ProgramData\Package Cache\{'  # NSIS "$TEMP" var Installers
```

```suggestion
              - 'C:\ProgramData\Package Cache\{'  # Microsoft Visual Redistributable installer  VC_redist/vcredist EXE
```
As the FP is not "NSIS" but "Microsoft Visual C++ 2015-2022 Redistributable" lauched by NSIS